publify_core/lib/publify_core/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module PublifyCore
-  VERSION = "9.1.0"
+  VERSION = "10.0.0"
```

Comment:
I prefer to bump the version separately when I make a release. Can you remove the PublifyCore version change?

Reply:
Sure.